riotfile.py

```diff
@@ -2482,32 +2617,18 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                         "langchain-cohere": latest,
                         "openai": latest,
                         "pinecone-client": latest,
-                        "botocore": latest,
-                        "cohere": latest,
-                    }
-                ),
-                Venv(
```

Comment:
I'd prefer to keep this extra venv testing the latest release of langchain, as we've been bit hard by surprise minor releases that include breaking changes.

Reply:
That's actually the situation that led me to remove it, so it's working as expected. Can we add it back in a separate pull request once the breaking version is supported?

Reply:
Will add this back in in #9902.